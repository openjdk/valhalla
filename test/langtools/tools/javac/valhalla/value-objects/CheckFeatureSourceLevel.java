/*
 * @test /nodynamiccopyright/
 * @bug 8279368
 * @summary Add parser support for value classes
 * @compile/fail/ref=CheckFeatureSourceLevel.out --release=13 -XDrawDiagnostics CheckFeatureSourceLevel.java
 */

public class CheckFeatureSourceLevel {

    static value class Value {
        public int v = 42;
    }
}
