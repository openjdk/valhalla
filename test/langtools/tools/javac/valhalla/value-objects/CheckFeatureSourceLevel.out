CheckFeatureSourceLevel.java:10:12: compiler.err.feature.not.supported.in.source.plural: (compiler.misc.feature.value.classes), 13, 18
1 error
